scripts/instaFeed.js

```diff
@@ -0,0 +1,44 @@
+async function instaFeedRequest(account) {
+    const posts = document.querySelector(".instaPosts")
+    posts.innerHTML = ""
+
+    const response = await fetch(`https://www.instagram.com/${account}/?__a=1`)
+        .then(x => x.json())
+        .then(json => {
```

Comment:
you should be able to remove 7-10 (and we don't want the log statement in any case)